features/appointment_type.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Appointment type
+  As Pension Wise
+  We want to offer tailored appointments to customers
+  So we can provide relevant information where possible
+
+  Scenario Outline: Appointment types
+    Given the customer is given the <type> appointment
+    When we send them their record of guidance
+    Then it should be a <variant> summary document
```

Comment:
Shall we stick with the term "record of guidance" and then make a future change to bring all the domain language inline? It's a bit confusing to use "record of guidance" and "summary document" interchangeably.